packages/devtools_app_shared/lib/src/service/service_manager.dart

```diff
@@ -666,7 +668,7 @@ class ServiceManager<T extends VmService> {
             dtd: dtdManager.connection.value,
           )
         : null;
-    _log.fine('rootPackageDirectoryForMainIsolate: $packageUriString');
+    _log.shout('rootPackageDirectoryForMainIsolate: $packageUriString');
```

Comment:
can you change this back to fine? We don't want this printing to console at the default log level

Reply:
Oh yes!